src/lib/rule-finder.js

```diff
@@ -57,21 +63,23 @@ function _getPluginRules(config) {
   const plugins = config.plugins;
   /* istanbul ignore else */
   if (plugins) {
-    plugins.forEach(plugin => {
-      const normalized = normalizePluginName(plugin);
-      const pluginConfig = require(normalized.module);
-      const rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules;
+    (ESLint.configType === 'eslintrc' ? plugins : Object.keys(plugins)).forEach(plugin => {
+      const normalized = ESLint.configType === 'eslintrc' ? normalizePluginName(plugin) : null;
+      const pluginConfig = normalized ? require(normalized.module) : null;
+      const rules = pluginConfig 
+        ? pluginConfig.rules === undefined ? {} : pluginConfig.rules
+        : plugins[plugin].rules === undefined ? {} : plugins[plugin].rules;
```

Comment:
```suggestion
      const rules = pluginConfig ? pluginConfig.rules : plugins[plugin].rules;
```
(if the change on line 74 is made)